client/src/components/promo/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { PromoSection, SubTitlePromo, Paragraph, TitlePromo } from './promo'
+import { Container } from '../../box/absoluteBox'
+import {WrapperColumn} from '../../box/flexBox'
+
+interface Props{
+    title: string;
+    description: string;
+    particle: string;
+    height: string;
+    background: string;
+}
+
+const Promo = ({title, description, particle, height, background}:Props) => {
+   
+    return (
+        <PromoSection ht={`${height}`+ 'px'} bphone={`${background}`}>
```

Comment:
```suggestion
        <PromoSection ht={`${height}`+ 'px'} bphone={background}>
```

Reply:
i did so it didn't work without `${}` 
I will do but it can not work

Reply:
It is working! Thank you. I didn't change in both places - and it didn;t work.